packages/ember-views/tests/views/view/remove_test.js

```diff
@@ -10,44 +9,8 @@ import viewKeyword from 'ember-htmlbars/keywords/view';
 import { objectAt } from 'ember-runtime/mixins/array';
 
 
-// .......................................................
-// removeChild()
```

Comment:
@rwjblue was this meant to be removed? If so, should `View#removeAllChildren` and friends have the same fate?